src/gui/mainwindow.cpp

```diff
@@ -1801,6 +1835,14 @@ void MainWindow::on_actionShowStatusbar_triggered()
     showStatusBar(isVisible);
 }
 
+void MainWindow::on_actionShowFiltersSidebar_triggered()
+{
+    const bool isVisible = static_cast<QAction *>(sender())->isChecked();
```

Comment:
Using `sender()` is discouraged. `QAction:::triggered()` has `bool checked` parameter so you could use it instead.